scripts/beesd@.service.in

```diff
@@ -56,4 +57,4 @@ AmbientCapabilities=CAP_DAC_OVERRIDE CAP_DAC_READ_SEARCH CAP_FOWNER CAP_SYS_ADMI
 NoNewPrivileges=true
 
 [Install]
-WantedBy=basic.target
+WantedBy=multi-user.target
```

Comment:
I'm not sure about this... While it may fix your problem (and isn't actually incorrect), I rather think that it solves your problem only by accident. `After=local-fs.target` should be enough for your lvm setup to become ready, otherwise we may need another `After=` line maybe?

Don't get me wrong: I think this change should go into the commit. But I think it doesn't reliably solve your problem. I think in case of lvm, there may be another synchronization point missing. What's the output of `systemd-analyze critical-chain`?